config/jobs/kubernetes-sigs/cluster-api/templates/cluster-api-periodics.yaml.tpl

```diff
@@ -274,4 +274,61 @@ periodics:
     testgrid-tab-name: capi-e2e-conformance-ci-latest-{{ ReplaceAll $.branch "." "-" }}
     testgrid-alert-email: sig-cluster-lifecycle-cluster-api-alerts@kubernetes.io
     testgrid-num-failures-to-alert: "4"
+- name: periodic-cluster-api-e2e-{{ ReplaceAll $.branch "." "-" }}
+  cluster: eks-prow-build-cluster
+  interval: {{ $.config.Interval }}
+  decorate: true
+  decoration_config:
+    timeout: 180m
+  rerun_auth_config:
+    github_team_slugs:
+    - org: kubernetes-sigs
+      slug: cluster-api-maintainers
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api
+    base_ref: {{ $.branch }}
+    path_alias: sigs.k8s.io/cluster-api
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+      - image: {{ $.config.TestImage }}
+        args:
+          - runner.sh
+          - "./scripts/ci-e2e.sh"
+        env:
+            # enable IPV6 in bootstrap image
+          - name: "DOCKER_IN_DOCKER_IPV6_ENABLED"
+            value: "true"
+          - name: GINKGO_SKIP
+            value: "\\[Conformance\\]"
+          - name: KUBERNETES_VERSION_MANAGEMENT
+            value: "ci/latest-1.31"
```

Comment:
[After rebase]

Let's try to generate these values based on the prowjob gen file

I think we should be able to calculate these values the same way as we do it here: `pull-cluster-api-e2e-upgrade-{{ ReplaceAll (last $.config.Upgrades).From "." "-" }}-{{ ReplaceAll (last $.config.Upgrades).To "." "-" }}-{{ ReplaceAll $.branch "." "-" }}`

KUBERNETES_VERSION_MANAGEMENT, KUBERNETES_VERSION. KUBERNETES_VERSION_UPGRADE_TO should be `{{ index (index $.versions ((last $.config.Upgrades).To)) "k8sRelease" }}`

KUBERNETES_VERSION_UPGRADE_FROM should be `{{ index (index $.versions ((last $.config.Upgrades).From)) "k8sRelease" }}`

Reply:
done